source/Calamari.Common/Variables/EnvironmentVariables.cs

```diff
@@ -0,0 +1,9 @@
+namespace Calamari.Common.Variables
+{
+    public static class EnvironmentVariables
+    {
+        public const string Prefix = "env:";
+        public static readonly string Id = "Octopus.Environment.Id";
```

Comment:
These are `DeploymentEnvironment` variables, where as the previous line is `Environment` or `RuntimeEnvironment` variables, same name, completely different things.